documentation/API_documentation/NumberVerificationdevice_phone_number_User_Story.md

```diff
@@ -0,0 +1,10 @@
+# Number Verification Device Phone Number API User Story
+
+| **Item** | **Details** |
+| ---- | ------- |
+| ***Summary*** | As an enterprise application developer, I want to retrieve the phone number associated with the line from which the call was made, so that I can ensure that I obtain the correct phone number and avoid identity theft fraud. |

```

Comment:
```suggestion
| ***Summary*** | As an enterprise application developer, I want to retrieve the phone number associated with the line from which the call was made, so that I can ensure that I obtain the correct phone number and avoid fraud e.g. identity theft. |

```